Kentor.AuthServices/IdentityProvider.cs

```diff
@@ -76,7 +77,7 @@ private void Validate()
                 throw new ConfigurationErrorsException("Missing binding configuration on Idp " + EntityId.Id + ".");
             }
 
-            if (SigningKey == null)
+            if (SigningKeys == null)

```

Comment:
As it's now a list, null check indeed became invalid. Needs to be changed to SigningKeys.Count == 0
Will do this on Monday as well.